.github/stale.yml

```diff
@@ -0,0 +1,63 @@
+# Configuration for probot-stale - https://github.com/probot/stale
+
+# Number of days of inactivity before an Issue or Pull Request becomes stale
+daysUntilStale: 30
+
+# Number of days of inactivity before an Issue or Pull Request with the stale label is closed.
+# Set to false to disable. If disabled, issues still need to be closed manually, but will remain marked as stale.
+daysUntilClose: false
+
+# Only issues or pull requests with all of these labels are check if stale. Defaults to `[]` (disabled)
+onlyLabels: []
+
+# Issues or Pull Requests with these labels will never be considered stale. Set to `[]` to disable
+exemptLabels:
+  - "approved"
+
+# Set to true to ignore issues in a project (defaults to false)
+exemptProjects: false
+
+# Set to true to ignore issues in a milestone (defaults to false)
+exemptMilestones: false
+
+# Set to true to ignore issues with an assignee (defaults to false)
+exemptAssignees: false
+
+# Label to use when marking as stale
+staleLabel: stale
+
+# Limit the number of actions per hour, from 1-30. Default is 30
+limitPerRun: 5
+
+# Comment to post when removing the stale label.
+# unmarkComment: >
+#   Your comment here.
+
+# Optionally, specify configuration settings that are specific to just 'issues' or 'pulls':
+pulls:
+  # Comment to post when marking as stale. Set to `false` to disable
+  markComment: >
+    Hello there,
+
+    We notice that this PR has been inactive for awhile now. It would be a waste of your time and effort if we just close this PR and
+    throw away the SVGs in here. Thus, 1 month from now, if this PR remains inactive/stale, we will fork your repo, fix any issues,
+    and merge your icons ourselves.
+
+    Since GitHub tracks contributions by commits, you will still be credited.
+    We also hope that you will help us merge this PR before the deadline comes.
+
+    Cheers,
+    Devicon Team
+  # Comment to post when closing a stale Pull Request.
+  closeComment: false
+issues:
+  # Comment to post when marking as stale. Set to `false` to disable
+  markComment: >
+    This issue has been automatically marked as stale because it has not had
+    recent activity. It will be closed if no further activity occurs. Thank you
+    for your contributions.
+  # Comment to post when closing a stale Issue.
+  closeComment: >
```

Comment:
I think this won't work because `daysUntilClose` is disabled; but if we enable it PR's will be closed too. 🤔